### ISSUE TITLE
Template: Letter (Request for Proposal, or "RFP") to video studios

### ISSUE THREAD

**Issue by ghost** [humanetech-community/humanetech-community-awareness#14]
Here is a template of a letter members working on a particular deliverable (here, a video) would send to potential movie studios (I am not talking about Century Fox here, but smaller studios!) to request a proposal and quotation. Having a template ensures consistency, structure and comprehensiveness in our approach to video studios, enabling any member to be involved in a coordinated fashion.

Note: For clarity, I used the example of the first proposed video of the Reality Shock campaign ("A Coffee Place").

## Template

[Name]
[Studio Name]

Dear [Name],

**Re. Request for proposal to produce a short video commissioned by the Humane Tech Community**

The Humane Tech Community ("HTC"), founded by The Center for Humane Technology (www.humanetech.com), is a grassroots movement whose purpose is to "Re-align Technology with Humanity's Best Interests". Based upon four pillars, namely Awareness, Regulation, Participate and Realization, members of the community are actively involved in numerous initiatives aimed at challenging the status quo.

One of these initiatives consists of structuring, organizing and managing several campaigns to raise public awareness of the potential pitfalls of rapid, drastic technology adoption and its significant impact on our society.

The first campaign, named "Reality Shock", involves the production and distribution of a series of short, dramatic videos (each a few minutes long) aimed at shocking the audience into introspection and reflection on the new hazards presented by technology (e.g. smartphone and social media addictions, privacy, social credits, etc) and how they may be personally affected.

We believe that to achieve maximum distribution and impact, these videos should be produced by a professional studio. We would like to invite your company to submit a detailed proposal for the production of one of these videos, with the working title "A Coffee Place".

HTC members will assess your proposal based on your preliminary ideas, relevant reference projects and budget.

As you will understand, the HTC as a community of volunteers will pay special attention to any proposal you can make with regard to budget, without sacrificing the desired quality of the final deliverable.

If your company sympathizes with our cause, or believes, as we do, that this project has a very high potential to be widely distributed and reach millions, we also invite you to consider working with us on a partnership basis, where you will essentially produce the video for free, in exchange for special credits and other advantages to be discussed. 

You will find in appendix the following information to help you prepare your proposal:

- Campaign theme and objectives
- Campaign description and objectives
- Video concept, script and objectives
- Scope of work
- Key issues
- Structure of your proposal

Should you have any query on this project, please contact anyone of the following HTC representatives:

- Contact 1
- Contact 2
- Contact 3

We hope you will find this project as exciting as we do and look forward to working with you.

Best regards,
[Name]
Humane Technology Community

#### "Mankind 2.0" Campaign Theme

Summary

Raising awareness of the impact of technology on our society by producing unsettling or jarring experiences resulting from a wide disparity between what was expected and what the real situation turns out to be.

Topics

- Smartphone addiction
- Social media addiction
- Privacy
- Social credits
- (More to come; all 'Harms of Technology' apply)

Objectives

- Shock the audience
- Lead to introspection and questioning
- Make the audience think and reach their own conclusions
- Provoke, rather than convince

Description

This theme is based on the following principles:

- Not to push a certain agenda, or lecture the target audience
- Instead, provoke introspection and lead the audience to reach their own conclusions as to whether they are comfortable with the impact of such and such aspect of technology on their lives
- Acknowledge that we have become a very different society in a very short time and ask the question: "Should we consider this more carefully?"

#### "Reality Shock" Campaign

Summary 

Series of short, dramatic/impactful videos that illustrate the pervasive use of technology and rapid, drastic changes in our society, leading to introspection and pondering the question: "Are we really fine with this?"

Goals

- Shock the audience
- Lead to introspection and questioning
- Make the audience think and reach their own conclusions
- Provoke, rather than convince

Audience

All ages from 16

Description

- Argumentation, reasoning and "lectures" only go so far, hence the pressing need to first trigger an "emotional" response.
- The campaign will feature short videos (2-3 minutes each) that aim to be dramatic and impactful through the combined use of words, cinematography and music.It does not aim to explicitly pursue a specific agenda, push on the audience a clear message, or impose a "lecture". 
- Rather, it aims to provoke introspection by telling provocative stories, the conclusions of which are left to the audience. 
- Why is it important to adopt a "neutral" approach based on providing factual observations in a spectacular and dramatic form? Because this campaign is based on the principle that we will not change our behaviors on the mere basis of facts and reason if we have no motivation to challenge our thinking in the first place and take a step back from our habits. The dramatic videos aim to shock and leave a strong impression that ideally will lead to wondering where each of us stands in all this.

#### Video concept and script

Summary

Short video depicting the rapid and drastic growth in smartphone addiction.

Goals

- Shock the audience into realizing that our world has changed so much and so quickly since the smartphone's advent.
- Contrast previous pastimes and customs with the now universal addiction to our devices.
- Get the audience to ponder whether we should be more than a bit concerned about such drastic changes to our society.

Concept

Let us look at how our pastimes have changed by considering a scene at a cafe that actually represents two decades. From leisurely and seemingly tranquil activities, we have become increasingly absorbed in our phones, in the process building the wealthiest and most powerful companies in the world’s history. At the end, we realize our phones have become our only source of gratification.

Script

The whole scene takes place in one of these coffee place terraces you can see everywhere in Europe. This kind of place used to be cliche for spending time leisurely with friends, reading newspapers or a book, and watching people.

The music starts slowly. Maybe only a few patrons (2 or 3, importantly most of them in their early twenties as this is not a cross-generational issue), all of them engaging in above-mentioned, "traditional" activities.

As the music gathers pace and grows crescendo, more and more patrons of all ages and all backgrounds (students, business people, etc), alone or in groups, sit in, some of them picking up their phones and getting busy. The music grows louder, more and more patrons arrive, this time already holding their phones as they arrive. We start superimposing data (in the form of these high-tech transparent windows) pertaining to each patron that is relevant for harvesting, and showing growing profits, market caps for tech companies, etc. 

Once the music finally calms down in the very last few seconds (just before 2:37), one person who was engrossed with his/her phone looks up from his/her screen, and realizes there is not a soul around.

We conclude with showing a question for the viewer on these few final notes.

Duration

Short, probably 2:37 mins. (determined by music).

Music

First 2:37 mins. of Beethoven’s Symphony 7 Movement 2 (“allegretto”)

#### Scope of work

- Propose finalized storyboard
- Identify and arrange shooting venue
- Arrange figurants and actors of all age groups (i.e. from 20 to 60)
- Propose cinematic effects to enhance video impact, in sync with proposed music theme
- After shooting, cooperate with HTC on the producing the final product

#### Key issues

- The script is simple given the proposed duration (2:37). It is on purpose as we want to impress a slow and steady progression (accentuated by the music going crescendo). Hence it will be key to enhance the appeal of the video by incorporating cinematic effects and/or suggesting additions to the script
- Of particular important, we want the video to be as "neutral" as possible (while obviously leading the audience to realize something feels wrong about the rapid changes by technology). This means for instance that actors engrossed with their phones should be viewed under a neutral light.

#### Structure of your proposal

- Introduction, confirming your understanding of our themes, concepts and importantly, objectives of our campaign and proposed video
- Your preliminary views on key issues (see above), comments on the script, and ideas as to cinematic effects
- Your preliminary views on the feasibility of arranging a shooting venue
- Your preliminary views on the the availability of actors/figurants (we may need 40-50).
- Your considerations on potential partnership (if of interest)
- Your rough budget (please avoid quoting a wide range)

**Comment by georgeshk** [comment]
Thanks @schrijver, it means a lot

**Comment by patmatsu** [comment]
Yes, very good, @georgeshk. To explain a bit about what I'm going to do with regard to the Academy for Creative Media: one of my good friends, Noah, is a good friend of an ACM graduate who is now living and working in the California film industry. My idea is to approach her through Noah to see what her ideas might be with regard to our proposal and, of course, to see if she wants to submit a bid herself. This is my general plan, which I'll discuss with Noah when I see him tomorrow.

**Comment by aschrijver** [comment]
(changed label to 'strategy' as this is about content/subject matter, not project/repo structure, workflow)

> If your company sympathizes with our cause, or believes, as we do, that this project has a very high potential to be widely distributed and reach millions, we also invite you to consider working with us on a partnership basis, where you will essentially produce the video for free, in exchange for publicity and other advantages to be discussed.

This requires special care and attention!

You know my position on openness, open-source, crowdsourcing, public domain and being an exemplar of humane tech. Before we can add text like this, this position has to be formalized and well-defined at the community level, however.

This does not preclude using texts similar to this in our current campaign plans, but probably involves leaving the 'partnership' and long-term 'publicity' parts out, or reformulating smartly (possible partnerships on the horizon, etc.)

Don't worry. There are enough rewards and benefits we can come up with, without implying a partnership, or special relationship (e.g. 'Special thanks go to ...' in credit roll + logo, 'Created with the help of ...', etc.). I suggest creating a separate issue to come up with sufficient beneficial offers, and linking to this issue.

Also we can make exceptions for our first campaigns, that do not imply a real partnership yet, like: 'This video was sponsored by ...'

**Comment by aschrijver** [comment]
I suggest creating a Pull Request with the template text above, so we can make reviewed correction, while continuing discussing here. The RFP can live as a `rfp-template.md` document in the `strategy` folder for now, and later moved. When text is stable, we'll create Word/OpenOffice/PDF derivatives with a version number.

**Comment by georgeshk** [comment]
> or reformulating smartly (possible partnerships on the horizon, etc.)

I believe if the studio agrees to produce for free, it is hard not to give them credits for the production (even when the studio is paid, credits should be paid to the producer).

I called it "publicity and other advantages to be discussed". If you do not like the word "publicity" as it may imply the studio is free to distribute the video through other means for their own benefits, thus in the process getting credit for a HTC initiative, I would propose to reword to "special credits and other advantages to be discussed", meaning they would have special recognition, but would not own the distribution rights (our video should be published on youtube under a HTC official account). It also leaves entirely open discussions on any other benefits. Agreed?

Would like to finalize this and not leave this important point of partnership for case by case discussions. I think it would be vague enough as proposed.

**Comment by georgeshk** [comment]
> I suggest creating a Pull Request with the template text above, so we can make reviewed correction, while continuing discussing here. 

Will do.

**Comment by aschrijver** [comment]
> Would like to finalize this and not leave this important point of partnership for case by case discussions. I think it would be vague enough as proposed.

As long as we'll take my point of care and attention into account, I am fine with that. But I wanted to mention it explicitly, so that we don't get these discussions when commitments have already been made.

**Comment by georgeshk** [comment]
Here it is in Word format: 
[RFP to videographers.docx](https://github.com/humanetech-community/humanetech-community-awareness/files/2477083/RFP.to.videographers.docx)

**Comment by patmatsu** [comment]
Thanks, @georgeshk. This is much better.

**Comment by georgeshk** [comment]
@patmatsu

Glad you like it!

**Comment by patmatsu** [comment]
A few questions:

- Should we include a request for a schedule or timeline? 

- Is it reasonable to assume that the "rough budget" includes the fees of actors, extras? We don't say who will be paying for their time, and this point may be ambiguous.

**Comment by patmatsu** [comment]
Is it possible to take over this issue--or to create a duplicate that I can work on?

**Comment by patmatsu** [comment]
Ok, thank you!

**Comment by aschrijver** [comment]
I have assigned this to you, @patmatsu 

I suggest checking the RFP document in `/stragegy` to see (i.e. [this document](https://github.com/humanetech-community/humanetech-community-awareness/blob/master/strategy/rfp-to-videographers.md) if everything in comments above is incorparated of the template, and then close this issue, and work on document improvements directly. You can then create Pull Requests (PR's) with your changes, which we can review.

**Comment by aschrijver** [comment]
Closing. RFP document is in PR review: https://github.com/humanetech-community/humanetech-community-awareness/pull/43
